### PR TITLE
Add a Cloud SQL sample

### PR DESCRIPTION
**Pull request by krwenholz**

### CODE REVIEW

File: examples/v2/sqladmin/python/sqladmin.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      'type': 'gcp-types/sqladmin-v1beta4:databases',
+      'metadata': {
+          'dependsOn': [instance_name]
+      },
+      'properties': {
+          'name': database_name,
+          'instance': instance_name,
+          'charset': 'utf8'
+      }
+  }]
```
Comment (adamharwayne) [inline examples/v2/sqladmin/python/sqladmin.py]: Add the delete-root action to match the jinja file.

File: examples/v2/sqladmin/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+## Deploy the template
+
+Use `sqladmin.yaml` to deploy this example template. When ready, deploy
+with the following command:
+
+    gcloud deployment-manager deployments create my-database --config sqladmin.yaml
+
+## Deleting the root user
+
+If you want to delete the root user, you can add the following resource.
```
Comment (adamharwayne) [inline examples/v2/sqladmin/README.md]: This is already in the jinja, I think it should be added to the python template as well. Documented in the overview and then this section can be removed.

Comment (krwenholz) [reply examples/v2/sqladmin/README.md]: Ah whoops. That was actually an accident, but it probably is a better practice to have folks delete the root user by default.

File: examples/v2/sqladmin/python/sqladmin.py (hunk 1 above)
Comment (krwenholz) [reply examples/v2/sqladmin/python/sqladmin.py]: Done.

**Pull request merged**